info/v1/metric.go

```diff
@@ -32,38 +32,21 @@ const (
 	MetricDelta = "delta"
 )
 
-// An exported metric.
-type Metric struct {
+// Spec for custom metric.
+type MetricSpec struct {
 	// The name of the metric.
 	Name string `json:"name"`
 
 	// Type of the metric.
 	Type MetricType `json:"type"`
-
-	// Metadata associated with this metric.
-	Labels map[string]string
```

Comment:
We spoke a bit offline, but labels allow us to identify different parameters of the same metric. For example, if I am exporting disk size as the "disk_size" metric and I have 2 disk: sda1 and sda2 I can export a label "disk_id=sda1" to specify these metrics are for that disk